test/forms/AddressUKFormProviderSpec.scala

```diff
@@ -36,7 +37,8 @@ class AddressUKFormProviderSpec extends StringFieldBehaviours {
     behave like fieldThatBindsValidData(
       form,
       fieldName,
-      stringsWithMaxLength(maxLength)
+      RegexpGen.from(apiAddressRegex),
+      errorToFind = Some(invalidKey)
```

Comment:
@ers2015112 We no need to pass errorToFind = Some(invalidKey), could you please look into it